server/graphql/v1/CollectiveInterface.js

```diff
@@ -500,6 +500,13 @@ export const CollectiveStatsType = new GraphQLObjectType({
           });
         },
       },
+      activeRecurringContributions: {
+        description: 'sums up all the active subscriptions',
+        type: GraphQLJSON,
+        resolve(collective) {
+          // TODO: invoke appropriate query methods
```

Comment:
What are the appropriate query methods to be invoked?

Reply:
You must implement the query in `context.loaders.Collective.stats` (https://github.com/opencollective/opencollective-api/blob/aad071630c5cb1d035ae757d711763defc036e3a/server/graphql/loaders/index.js#L64). Check the other loaders to see how to do that

Reply:
It would be great if you could give me a bit more of an insight since I'm not that familiar with the codebase.

Reply:
If you're not familiar with GraphQL loaders I suggest getting a quick look at https://github.com/graphql/dataloader.
The new loader must be implemented in `server/graphql/loaders/index.js` > `context.loaders.Collective.stats`: https://github.com/opencollective/opencollective-api/blob/aad071630c5cb1d035ae757d711763defc036e3a/server/graphql/loaders/index.js#L64

This loader should query the collective's active subscriptions by looking at the `Orders` table (interval NOT NULL, status=ACTIVE) and sum the `totalAmount`, grouping results by `interval`. This will provide the required data for `activeRecurringContributions`.